.gitignore

```diff
@@ -2,3 +2,4 @@ doc/*.html
 __pycache__
 *.swp
 unittests.log
+*~
```

Comment:
Come to think of it, where does '*.swp' and 'unittests.log' get created?

Reply:
For `unittests.log`, I often capture my unit test output to a log file of that name, so I figured ignoring it would avoid my accidentally checking it in.

As for `*.swp`, those show up when editing via the "other" editor (that is not named emacs).  When they show up as files that are not tracked, again, it is an attempt to be sure they don't get checked in accidentally.

Finally, the `*~` occurs from emacs backup files.

Does ignoring these three items cause a problem?

Reply:
I don't know how I was misreading the '*~" one, but it finally dawned on me on the way home tonight - d'oh! 
No problem -  I was just wondering (and being thick as a brick about it ...)